spec/javascripts/components/story/story_item_spec.js

```diff
@@ -80,4 +80,38 @@ describe('<StoryItem />', () => {
       expect(children).toHaveProp('title', title);
     });
   });
+
+  describe('when the story is accepted', () => {
+    const story = storyFactory({
```

Comment:
We should do this inside the beforeEach as well :smile: Let's avoid object creation and any logic we may need to debug outside of `beforeEach`, `afterEach` and `it`